src/main/java/io/swagger/client/auth/OAuthFlow.java

```diff
@@ -23,18 +23,11 @@
  * If needed, contact us to obtain a release under GPL Version 2 or 3
  * or a different license than the AGPL.
  */
-package org.ow2.proactive.catalogclient.service;
+package io.swagger.client.auth;
 
-/**
- * This class enables to get the service beans in order to be used from an outside configuration.
- * The beans enable to access to the service thanks to this class.
- */
-public final class CatalogClientLib {
-
-    private CatalogClientLib() {
-    }
-
-    public static CatalogObjectService getCatalogObjectService() {
-        return CatalogObjectService.getInstance();
-    }
+public enum OAuthFlow {
+    accessCode,
+    implicit,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)